lib/runner/index.js

```diff
@@ -49,6 +49,7 @@ _.assign(Runner.prototype, {
             data: Runner.normaliseIterationData(options.data, options.iterationCount),
             environment: options.environment,
             globals: _.has(options, 'globals') ? options.globals : this.options.globals,
+            collectionVariables: collection.variables,
```

Comment:
Could you add a @todo to move this to item level, so we can support Item and ItemGroup variables?